Team9Project/src/com/indragie/cmput301as1/Destination.java

```diff
@@ -128,6 +114,8 @@ public void setLocation(double latitude, double longitude) {
 	public int hashCode() {
 		final int prime = 31;
 		int result = 1;
+		result = prime * result
+				+ ((location == null) ? 0 : location.hashCode());
 		result = prime * result + ((name == null) ? 0 : name.hashCode());
 		result = prime * result
```

Comment:
These were all automatically generated by Eclipse, I think it might be automatically splitting them into multiple lines because they're long? Even if I do fix them, they'll just go back to that once they're regenerated :/